eslint.config.mjs

```diff
@@ -0,0 +1,30 @@
+import globals from 'globals'
+import pluginJs from '@eslint/js'
+import stylisticJs from '@stylistic/eslint-plugin-js'
+
+export default [
+  { files: ['**/*.js'], languageOptions: { sourceType: 'commonjs' } },
+  { languageOptions: { globals: globals.node } },
+  pluginJs.configs.recommended,
+  {
+    plugins: {
+      '@stylistic/js': stylisticJs
+    },
+    ignores: ['dist', 'node_modules'],
```

Comment:
having the Ignores like this for me didn't work, but having them in their own object at line 8, before pluginJs.configs.recommended did the trick. Following this format:
{ ignores: ['**/dist/*', '**/node_modules/*'] },

Reply:
Thanks for pointing this out, just changed it to { ignores: [ 'dist' ] }. node_modules is actually not required to explicitly ignore as it is ignored by default